modules/persistent-storage-csi-vsphere-rwx.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-rwx_{context}"]
+= ReadWriteMany vSphere volume support
+
+If the underlying vSphere environment supports the vSAN file service, then vSphere Container Storage Interface (CSI) Driver Operator installed by
+{product-title} supports provisioning of ReadWriteMany (RWX) volumes. If vSAN file service is not configured, then ReadWriteOnce (RWO) is the only access mode available. If you do not have vSAN file service configured, and you request RWX, the volume fails to get created and an error is logged.
```

Comment:
This behavior means that configurations shown in our example work for pods with 1 replica (or where only one pod mounts this volume). However upon trying to mount a second pod; the user sees an error (which isn't expected; given the 'accessMode's' configuration). Isn't this going to be 'confusing' looking at the configuration (it should start; right - that's how it works for EVERY other RWX volumeClaim)?

Reply:
No, if you ask for `ReadWriteMany` volume type and underlying backend does not support it, regardless of how many pods are using it - volume provisioning won't work.

Reply:
Ack